hw03/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+/* hw03: Functions 
+Task 1 - Напишіть функцію яка рекурсивно буде знаходити суму всіх непарних додатніх чисел до якогось числа.
+*/
+
+function recursiveOddSumTo(number) {
+	if (number <= 0) return 0;
+
+	if (number % 2 !== 0) {
+		return number + recursiveOddSumTo(number - 1);
+	} else {
+		return 0 + recursiveOddSumTo(number - 1);
+	}
+}
+
+console.log('recursiveOddSumTo= ' + recursiveOddSumTo(1)); // 1
+console.log('recursiveOddSumTo= ' + recursiveOddSumTo(10)); // 25
+
+/*
+Task 2 - Напишіть функцію яка ітеративно (в циклі) буде знаходити суму всіх непарних додатніх чисел до якогось числа.
+*/
+
+function iterativeOddSumTo(number) {
```

Comment:
```suggestion
function iterativeOddSumTo(limit) {
```
Більш вдала назва параметра, можна подумати над своїм варіантом

Reply:
те що поза блоком // ваш код тут, також можна змінювати якщо є кращі варіанти, головне не змінювати перевірку коду, тобто виклик з аргументами